Content.Server/StationEvents/BasicStationEventSchedulerSystem.cs

```diff
@@ -57,8 +57,8 @@ public override void Update(float frameTime)
         /// </summary>
         private void ResetTimer(BasicStationEventSchedulerComponent component)
         {
-            // 5 - 25 minutes. TG does 3-10 but that's pretty frequent
-            component.TimeUntilNextEvent = _random.Next(300, 1500);
+            // 15-35 minutes
+            component.TimeUntilNextEvent = _random.Next(900, 2100); // Parkstation-SlowEvents
```

Comment:
This fucking sucks, I can't believe this works like this.

https://github.com/space-wizards/space-station-14/pull/25959